server/webhook.go

```diff
@@ -3,37 +3,48 @@ package main
 import (
 	"context"
 	"crypto/hmac"
-	"crypto/sha1"
+	"crypto/sha1" //nolint:gosec
```

Comment:
Otherwise `golangci-lint` will complain that an insecure cypher is used.

Reply:
We should seriously consider moving to a stronger hash algorithm.

Reply:
I don't think we can do this. As https://developer.github.com/webhooks/ documents GitHub is using a HMAC with sha1 to generate the signature. No support for something else.

Reply:
Ah thanks. Perhaps a comment to explain it might be useful to the readers.

Reply:
Done in crypto/sha1. Great suggestion :+1: